src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -6,6 +6,16 @@
  void npe(String a, String b) {
   if (a == null) {
    System.out.println();
+   System.out.println();
+  } else {
+
+  }
+  a.length();
+ }
+
+ void npe2(String a, String b) {
+  if (a == null) {
+   System.out.println();
   } else {
```

Comment:
Reporter: PMD
Rule: EmptyIfStmt
Severity: WARN
File: /home/travis/build/tomasbjerre/violations-test/src/main/java/se/bjurr/violations/lib/example/MyClass.java L19
 

Empty Code http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/empty.html#EmptyIfStmt

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>